src/vocation.cpp

```diff
@@ -109,6 +109,10 @@ bool Vocations::loadFromXml()
 			voc.fromVocation = pugi::cast<uint32_t>(attr.value());
 		}
 
+		if ((attr = vocationNode.attribute("lessloss"))) {
```

Comment:
Well you added the xml loading part, but nothing happens with the code.
Did you miss uploading some code?